app/forms/feedback_form.rb

```diff
@@ -0,0 +1,31 @@
+class FeedbackForm
+  include ActiveModel::Model
+
+  attr_accessor :name, :email, :message, :booking_type
+
+  validates :name, presence: true
+  validates :email, presence: true, format: { with: /.+@.+\..+/ }
+  validates :message, presence: true
+  validates :booking_type, inclusion: { in: %w(online_booking pension_type_tool) }
```

Comment:
Would consider renaming `booking_type` to something else – not sure the pension type tool is that kind of thing.

Not quite sure what it should be called though!